platform-service-impl/src/main/java/esa/mo/platform/impl/provider/opssat/AutonomousADCSOPSSATAdapter.java

```diff
@@ -59,14 +62,18 @@ public class AutonomousADCSOPSSATAdapter implements AutonomousADCSAdapterInterfa
 
   private static final float MAX_REACTION_WHEEL_SPEED = 1047.197551f;
   private static final float MAX_REACTION_WHEEL_TORQUE = 0.0001f;
-  private static final int IADCS_WATCH_PERIOD_MS = 10 * 1000;
-  private static final int IADCS_INIT_BACKOFF_MS = 10 * 1000;
-  private static final int INIT_FAILED_STOP_THRESHOLD = 3;
+  private int iADCS_watch_period_MS = 10 * 1000;
```

Comment:
Use lowerCamelCase for the variable names in Java instead of snake_case.
Use MACRO_CASE for consts.
See the difference:
https://en.wikipedia.org/wiki/Naming_convention_(programming)#Examples_of_multiple-word_identifier_formats